setup.py

```diff
@@ -27,15 +27,15 @@
     version += ".post" + str(int(time.time()))
 
 install_requires = [
-    "pandas>1.5.2,<2.1",  # requires numpy
+    "pandas>1.5.2,<1.6",  # requires numpy
```

Comment:
ufff what happens with pandas 2, does it break anything?

Since with probability 99% the answer is yes XD, would it be an easy fix or it will take some time?

Reply:
pandas 2 removes DataFrame.append() which is used in scrapbook.models:
https://github.com/nteract/scrapbook/blob/78f28ddd170528440872f88b77fcf30c7a10e422/scrapbook/models.py#L255

Reply:
@SimonYansenZhao is there a workaround or another way for scrapbook to get the information of the notebook?